### PR TITLE
CC Dedicated New Pricing and Machine types

### PR DESCRIPTION
**Pull request by gemma-shay**
Updated pricing and IOPS to the new configuration.
Added a note that existing clusters will be transitioned automatically by end of December.

Closes https://cockroachlabs.atlassian.net/browse/CC-5412

### CODE REVIEW

**Review by jramos-crl — APPROVED** [review]
Looks good to me! @gemma-shay

**Comment by gemma-shay** [comment]
@ianjevans do you have time for a quick review? hoping to get this out tomorrow

File: _includes/cockroachcloud/cockroachcloud-pricing.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
- Hardware configuration            | AWS IOPS                        | GCP Pricing (per node, per hour)	  | AWS Pricing (per node, per hour) 
------------------------------------+---------------------------------+------------------------------------+------------------------------------
-`Option 1` (2 vCPU, 60 GiB disk)   | 600	                           | $0.50	                            | $0.55
-`Option 2` (4 vCPU, 150 GiB disk)  | 900	                           | $0.89                            	| $1.02
-`Option 3` (8 vCPU, 500 GiB disk)  | 1800	                           | $1.78                              | $2.00
-`Option 4` (16 vCPU, 900 GiB disk) | 2000	                           | $3.38                              | $3.83
+ Hardware configuration            | GCP Pricing (per node, per hour)	  | AWS Pricing (per node, per hour) 
+-----------------------------------+------------------------------------+------------------------------------
+`Option 1` (2 vCPU, 60 GiB disk)   | $0.50	                            | $0.55
```
Comment (ianjevans) [inline _includes/cockroachcloud/cockroachcloud-pricing.md:5]: nit: the tab spacing is off for AWS pricing on this row.

**Review by ianjevans — APPROVED** [review]
LGTM, just a small nit about tab spacing.

File: _includes/cockroachcloud/cockroachcloud-pricing.md (hunk 1 above)
Comment (gemma-shay) [reply _includes/cockroachcloud/cockroachcloud-pricing.md]: can't see the difference in atom for some reason, hopefully this fixes it

**Pull request merged**